js/blocks.js

```diff
@@ -3616,8 +3616,8 @@ function Blocks(canvas, stage, refreshCanvas, trashcan, updateStage, getStageSca
         if (myBlock.name === 'start' || myBlock.name === 'drum') {
             turtle = myBlock.value;
             var turtleNotInTrash = 0;
-            for(var i = 0; i < this.turtles.turtleList.length; i++) {
-                if(!blocks.turtles.turtleList[i].trash) {
```

Comment:
@walterbender It's a cosmetical change. I does work with blocks too, I used `this` to keep a convention. Line 3631 (`return;`) is crucial for this patch.